tuxemon/states/combat/combat.py

```diff
@@ -152,6 +154,8 @@ def load_technique_animation(technique: Technique) -> Optional[Sprite]:
             image = graphics.load_and_scale(fn)
             images.append((image, frame_time))
         tech = SurfaceAnimation(images, False)
+        if fix_direction:
+            tech.flip(True)
```

Comment:
Note that some animations will need to be flipped in the y axis too.

Reply:
To be honest, I'm not super happy with the current implementation, because it allows flipping only in X `or` Y. Should we support flipping in both directions?

Reply:
> To be honest, I'm not super happy with the current implementation, because it allows flipping only in X or Y. Should we support flipping in both directions?

Yes

> Another question, I understand the various "front" animations should support the "up" mode, correct?

?

Reply:
The "front" animations are facing down so if we would want to flip them in Y-axis, that would be equivalent to the "up" animations.

Reply:
Not sure about this.